issues-policy.md

```diff
@@ -0,0 +1,84 @@
+- What priority is the issue?
+- What area manages this issue?
+
+If these questions can't be answered in the initial examination, we ask clarifying questions in the comments.
+
+There are types of issues that are closed during the diagnosis and triage phase:
+
+- **kudos**: Express thanks, and close the issue.
+- **product issue**: Issues that request product changes are closed. We may also take additional actions, as described below.
+- **CoC violations**: These issues are closed, and reported if the [CoC violation](https://dotnetfoundation.org/code-of-conduct) merits reporting and blocking.
+- **duplicates**: Duplicates are closed with a comment referencing the existing issue.
+- **doc-ok**: The customer is incorrect, and the doc is correct.
+- **forum**: Issues that are support or forum requests are directed to Stack Overflow, or other support sites, and closed.
+
+### Actions on product issues
+
+Depending on the nature of the product issue, we may choose to:
+
+- Transfer the issue to the appropriate product repository.
+- Close the issue as a duplicate of existing product requests.
+- Close the issue with a recommendation to open on the product repository.
+
+Evaluating the correct course of action is subjective. The team members use their judgment on the correct action.
+
+### Actions on content issues
+
+For other issues, the team:
+
+- assigns a priority
+- assigns a milestone, usually "Backlog"
+- reflects tier level support
+- assesses if an issue is a good "up for grabs" issue.
+
+Priority levels are based on the following guidelines, but are subjective. The milestones are also subjective, and will be based on other priorities such as current product release schedules and upcoming launches.
+
+- **P0**: A docs omission or error prevents customers from succeeding in a common scenario.
+  -  ***P0** issues are addressed in the current sprint, taking precedence over previously scheduled work.
+- **P1**: A docs omission or error makes a common scenario much harder, or blocks other well-known scenarios.
+  - **P1** issues are scheduled in a timely manner. Often, P1 issues are planned for an upcoming milestone.
+- **P2**: Issues that cause minor inconveniences, or affect a low page view article.
+  - **P2** issues are generally fixed when an article is updated for higher-priority reasons.
+- **P3**: Issues that are requests for edge case scenarios.
+  - **P3** issues are placed in the backlog and are considered for update when articles are updated for higher-priority reasons.
+
+The diagnosis and triage activities are time-boxed to enable the team to make progress on tasks scheduled for each sprint. Each team member spends at most 30 minutes per day in diagnosis and triage. In order to stay current, diagnosis and triage cannot turn into a resolution session. At most, consider if a single sentence or two can answer a question and provide enough insight for a customer PR.
```

Comment:
I'm not sure who the last sentence is directed at. The person opening a PR should consider? We should consider?

Reply:
>The diagnosis and triage activities are time-boxed to enable the team to make progress on tasks scheduled for each sprint.

Rewording suggestion:

> Team members spend a limited amount of time on diagnosis and triage so they can make progress on tasks scheduled for each sprint.

3rd and 4th sentences seem directed at us and could be removed from here.

Reply:
I followed @tdykstra 's suggestion and removed the last two sentences.